JSTests/stress/intl-locale.js

```diff
@@ -152,7 +157,7 @@ shouldBe(
         ? 'en-Latn-US-1abc-foobar-variant-a-aa-aaa-u-kn-x-reserved'
         : 'en-Latn-US-variant-foobar-1abc-a-aa-aaa-u-kn-x-reserved'
 );
-shouldBe(new Intl.Locale('cel-gaulish', { script: 'Arab', numberingSystem: 'gujr' }).toString(), $vm.icuVersion() >= 69 ? 'xtg-Arab-u-nu-gujr' : 'xtg-Arab-u-nu-gujr-x-cel-gaulish');
+shouldBeOneOfThem(new Intl.Locale('cel-gaulish', { script: 'Arab', numberingSystem: 'gujr' }).toString(), [ 'xtg-Arab-u-nu-gujr', 'xtg-Arab-u-nu-gujr-x-cel-gaulish', 'cel-Arab-gaulish-u-nu-gujr' ]);
```

Comment:
This one is noteworthy because the result we see on Fedora, 'cel-Arab-gaulish-u-nu-gujr', does not match either of the previous expected results.